### PR TITLE
Fix broken links to 1.14 docs version that is no longer supported on the website

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Since the 1.14 version of the documentation is no longer supported on the website, we need to change the links to the 1.14 version of docs stored in GitHub.

Changes proposed in this pull request:

- Fix broken links to 1.14 docs version

**Related issue(s)**

### CODE REVIEW

**Review by alexandra-simeonova — APPROVED** [review]
🌟

File: content/blog-posts/2020-08-06-release-notes-1.14/index.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ We upgraded Grafana to v7.0 which brings a lot of improvements like a new panel
...
+We continue our work on the Serverless Runtime for Kyma. In this release, we migrated from Knative Serving to pure Kubernetes resources. In its new shape and form, Serverless is more lightweight, doesn't require any additional components, and provides more control over underlying resources. With this change, Serverless uses only Jobs, Deployments, Services, and Horizontal Pod Autoscalers. To see how all these pieces fit together, take a look at the [Serverless architecture](https://github.com/kyma-project/kyma/blob/release-1.14/docs/serverless/02-01-serverless.md). Importantly, the whole migration from the previous solution to the current one is fully automated and executed during Kyma update.
 
 ### Function CRD validation
```
Comment (majakurcius) [inline content/blog-posts/2020-08-06-release-notes-1.14/index.md:133]: The line below (134) needs to have the link switched as well, to:
> We introduced the validation of Function CRDs. Every new version of the Function CRD is verified by the [defaulting and validation webhooks](https://github.com/kyma-project/kyma/blob/release-1.14/docs/serverless/03-01-supported-webhooks.md) before you apply it on your cluster. Validation works not only in the UI but also in the terminal when you apply resources using kubectl.

Comment (klaudiagrz) [reply content/blog-posts/2020-08-06-release-notes-1.14/index.md:133]: I overlooked this one, thanks! 🙂

**Comment by klaudiagrz** [comment]
Due to my error, I have to close this PR and open a [new one](https://github.com/kyma-project/website/pull/661/files) 😅

**Pull request closed without merging**